libfwupdplugin/fu-udev-device.c

```diff
@@ -442,6 +463,17 @@ fu_udev_device_probe(FuDevice *device, GError **error)
 		}
 	}
 
+	/* if the device is a dGPU try to fetch it from vbios_version */
+	if (g_strcmp0(priv->subsystem, "pci") == 0 && priv->class == 0x30000 &&
```

Comment:
are the device classes enumerated? It seems like something we should have as a #define somewhere.

Reply:
added